docs/man/django-admin.1

```diff
@@ -1916,7 +1916,7 @@ correctly:
 .sp
 .nf
 .ft C
-$ pip install tblib
```

Comment:
Man page is generated, so I will revert this change (see [django-admin-man-page](https://docs.djangoproject.com/en/stable/internals/contributing/writing-documentation/#django-admin-man-page)).